### PR TITLE
Feat/share token

### PR DESCRIPTION
**Pull request by prakash-bhatt-rs**

### CODE REVIEW

**Comment by prakash-bhatt-rs** [comment]
#416

File: ssr/src/page/token/info.rs [hunk 1]
```diff
@@ -64,12 +81,19 @@ fn TokenInfoInner(meta: TokenMetadata) -> impl IntoView {
...
+                            >
+                            <Icon icon=icondata::AiShareAltOutlined/>
+
+                            </button>
+                            // <div class="p-1 bg-white/15 rounded-full">
+                            //     <Icon
+                            //         class="text-sm md:text-base text-white"
+                            //         icon=icondata::BsArrowUpRight
+                            //     />
+                            // </div>
```
Comment (ravibazz) [inline ssr/src/page/token/info.rs:96]: Should we remove the commented code?

File: ssr/src/page/wallet/tokens.rs [hunk 2]
```diff
@@ -114,9 +133,17 @@ pub fn TokenView(
...
+                                    on:click=move|_| share_profile_url()
+                                    class="text-white text-center p-1 text-lg md:text-xl bg-primary-600 rounded-full"
+                                    >
+                                    <Icon icon=icondata::AiShareAltOutlined/>
+
+                                </button>
+                                // <div class="flex items-center justify-center w-8 h-8 bg-white/15 rounded-full">
+
+                                //     <Icon icon=icondata::BsSend/>
+                                // </div>
```
Comment (ravibazz) [inline ssr/src/page/wallet/tokens.rs:146]: Lets remove the commented code

**Review by ravibazz — CHANGES_REQUESTED** [review]

File: ssr/src/page/wallet/tokens.rs (hunk 2 above)
Comment (prakash-bhatt-rs) [reply ssr/src/page/wallet/tokens.rs:146]: Okay.

File: ssr/src/page/wallet/share_token_popup.rs [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+        </div>
+        <button on:click=move|_| close_popup.set(true)
+        class="py-4 w-3/4 text-lg text-center text-white rounded-full bg-primary-600"
+    >
+    Back
+    </button>
+
+        </div>
+    }
+}
```
Comment (ravibazz) [inline ssr/src/page/wallet/share_token_popup.rs:47]: Some of the components in this file looks reusable. Should be moved inside components folder?

**Review by ravibazz — CHANGES_REQUESTED** [review]
Changes looks good just need to move some of the components to components folder as they can be reused in future in other pages.

File: ssr/src/component/share_popup.rs [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+#[component]
+fn ProfileLoading() -> impl IntoView {
+    view! {
+        <div class="rounded-full animate-pulse basis-4/12 aspect-square overflow-clip bg-white/20"></div>
+        <div class="flex flex-col gap-2 animate-pulse basis-8/12">
+            <div class="w-full h-4 rounded-full bg-white/20"></div>
+            <div class="w-full h-4 rounded-full bg-white/20"></div>
+        </div>
+    }
+}
```
Comment (ravibazz) [inline ssr/src/component/share_popup.rs:171]: Why is this needed?

**Review by ravibazz — APPROVED** [review]

**Pull request merged**